proc_IR_noecho.py

```diff
@@ -44,12 +44,9 @@
     s /= ph0
     #}}}
     s.ft('t2')
-    print(ndshape(s))
-    print(np.sign(s['t2',2047]['indirect',15]))
-    s *= -1    
```

Comment:
this hard-coded stuff is taken care of in the `recovery` and `decay` functions in an automatic way, now

Reply:
Please be sure to go look at those and see how they work, and hopefully shed light on what I meant in our slack conversation